config/logger.js

```diff
@@ -1,17 +1,29 @@
 import { serverEnv } from "@config/schemas/serverSchema";
 import * as PinoLogger from "pino";
 
-const config = {
+const development = {
   transport: {
     target: "pino-pretty",
     options: {
+      level:'debug',
+      colorize: true,
+      translateTime:"SYS:dd-mm-yyyy HH:MM:ss",
+      ignore:"pid,hostname"   //Ignores process id and hostname
+    },
+  },
+};
+const production = {
+  transport: {
+    target: "pino-pretty",
+    options: {
+      level:'info',
       colorize: true,
     },
   },
 };
 
 let logger;
 
-logger = PinoLogger.pino(serverEnv.NODE_ENV === "development" ? config : {});
+logger = PinoLogger.pino(serverEnv.NODE_ENV === "development" ? development : production);
 
-export default logger;
+export default logger;
```

Comment:
Extra blank line is recommended by GitHub, as shown in red

```suggestion
export default logger;

```